src/main/java/com/thevoxelbox/voxelsniper/performer/type/combo/ComboComboNoPhysicsPerformer.java

```diff
@@ -19,10 +21,9 @@ public class ComboComboNoPhysicsPerformer extends AbstractPerformer {
 
     @CommandMethod("")
```

Comment:
I might be wrong. Do you need that here?

Reply:
It's fine, it means "no addition" to the global class command annotation above ``performer|perf|p combo-combo|cc``. This corresponds to the handler for the ``CommandMethod`` declared on the class, it acts as a prefix for internal ``CommandMethod``.